src/Iter.mo

```diff
@@ -216,16 +248,16 @@ module {
   /// assert(?3 == iter.next());
   /// assert(null == iter.next());
   /// ```
-  public func fromArray<T>(xs : [T]) : Iter<T> {
-    var ix : Nat = 0;
-    let size = xs.size();
+  public func fromArray<T>(array : [T]) : Iter<T> {
```

Comment:
I'm not sure we should worry about but both fromArray and fromVarArray have (and always have had) a small space leak, because the iterator will hold on to the source array even after iteration has finished. @luc-blaeser is this worth fixing or should we just live with it?